app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = %w[moise rushanika Nganulo]
```

Comment:
- [ ] Let's update the method from    `@users = %w[moise rushanika Nganulo] `to   `@users = User.all`